pkgs/development/python-modules/paddlepaddle/default.nix

```diff
@@ -0,0 +1,74 @@
+{ stdenv
+, lib
+, buildPythonPackage
+, fetchPypi
+, python
+, pythonOlder
+, pythonAtLeast
+, autoPatchelfHook
+, openssl_1_1
+, zlib
+, setuptools
+# runtime dependencies
+, httpx
+, numpy
+, protobuf
+, pillow
+, decorator
+, astor
+, paddle-bfloat
+, opt-einsum
+}:
+
+let
+  pname = "paddlepaddle";
+  version = "2.5.0";
+  format = "wheel";
+  pyShortVersion = "cp${builtins.replaceStrings ["."] [""] python.pythonVersion}";
+  hashAndPlatform = import ./binary-hashes.nix { pythonVersion = pyShortVersion; system = stdenv.system; };
+  src = fetchPypi ({
+    inherit pname version format;
+    dist = pyShortVersion;
+    python = pyShortVersion;
+    abi = pyShortVersion;
+  } // hashAndPlatform);
+in
+buildPythonPackage {
+  inherit pname version format src;
+
+  disabled = pythonOlder "3.9" || pythonAtLeast "3.11";
+
+  nativeBuildInputs = lib.optionals stdenv.isx86_64 [
```

Comment:
```suggestion
  nativeBuildInputs = lib.optionals stdenv.isLinux [
```

Reply:
In NixOS, we need to use `patchelf` to patch the hardcoded library paths of the binary.

Reply:
:+1: I added it to my first commit.